### ISSUE TITLE
Forwarding properties directly breaks rendering process

### ISSUE THREAD

**Issue by maticzav** [vadimdemedes/ink#106]
Hey 👋,

Thank you for building a new version of `Ink`. I am rewriting Emma entirely and wanted to use the second version since it's a lot easier to gain typescript support in combination with React (ps.: let me know if you want me to make type definitions PR).

```js
import React from 'react'
import { render, Color, Box } from 'ink'

class Counter extends React.Component {
  constructor(props) {
    super(props)

    this.state = {
      i: 0,
    }

    this.handleChange = this.handleChange.bind(this)
  }

  handleChange(query) {
    this.setState({})
  }

  render() {
    // <Foo value={`a ${this.state.i}`} />
    return (
      <Box>
        <Box marginRight={1}>Enter your query:</Box>

        <Color>{this.state.i} tests</Color>
        {/* <Foo value={`${this.state.i} tests`} /> */}
        <Bar value={`${this.state.i} tests`} />
      </Box>
    )
  }

  componentDidMount() {
    this.timer = setInterval(() => {
      this.setState({
        i: this.state.i + 1,
      })
    }, 100)
  }

  componentWillUnmount() {
    clearInterval(this.timer)
  }
}

const Foo = ({ value }) => <Color>{value} a</Color>
const Bar = ({ value }) => <Color>{value}</Color>

render(<Counter />)
```

I hope I haven't missed any TypeScript definition in the code above. Regardless, if we use the `Bar` component instead of `Foo` the output results in duplicated strings. I suppose you've also found this in `ink-text-input`.

Let me know if I missed anything 🙂

**Comment by vadimdemedes** [comment]
Hey! Yep, that's a known issue, it's a regression from https://github.com/vadimdemedes/ink/commit/a8be56251f9e678af6efcd88c6b54f027395cc16, reverting and releasing now ;)

Re TS definitions, thanks, but I want to hold off on this until I'm sure that API won't change (after all there's not official 2.0 release yet).

**Comment by vadimdemedes** [comment]
Released `2.0.0-9`.
